### ISSUE TITLE
Settings screen for opting out of experiments

### ISSUE THREAD

**Issue by jhugman** [mozilla-mobile/fenix#16213]
The Nimbus project needs a preference screen for the user to find out about studies that they have been involved in.

This issue requires that the user have a global opt-out, i.e. to be not involved in any studies/experiments.

The Nimbus team is happy to do this work, but may require some UX review, or design help.

**Comment by lime124** [comment]
hey @jhugman - unfortunately my whole team is just embarking on a design sprint that will last well into December. At the very least, I can help by reviewing and providing feedback. Can you help me understand a bit more about the requirements? Who will see this and what sorts of things do they need to see/interact with? Thanks!

**Comment by AnaMedinac** [comment]
@lime124 Hey Tif! 

As part of the legal requirements we need to allow users to be able to globally opt-out of experiments. They can currently do so from Settings --> Data Collection --> Experiments (turning on/off the toggle). For 2020 and our initial milestone, the legal/policy team is ok with the current set up, so we don't need any new UX for this. 

However, in the future, post 2020, legal/policy team is asking us to implement a about:studies equivalent on mobile, where users can identify all the experiments they are and were involved in. I'm going to start working on the requirements in the upcoming week (https://jira.mozilla.com/browse/SDK-130), to define the things that users need to see/interact with. We most likely will need some UX support for this screen in early 2021 if possible.

**Comment by eliserichards** [comment]
Designs: https://www.figma.com/file/bZyKtDQECrnkiW6uhGxlwG/Nimbus?node-id=104%3A17984

**Comment by gabrielluong** [comment]
The relevant Nimbus API is the `getActiveExperiment`, which will return the user's enrolled experiment, and `optOut`.

**Comment by brampitoyo** [comment]
@gabrielluong I’ve added the designs for empty states in the same Figma file.

TL;DR – Show each title on its own without content under it. This approach will make sure that our design and behaviour aligns with `about:studies` on Desktop.

**Comment by Amejia481** [comment]
We could close this as it was covered on https://github.com/mozilla-mobile/fenix/pull/20765
